### ISSUE TITLE
How to use conda to update pytorch to 0.4 version

### ISSUE THREAD

**Issue by AlexHex7** [pytorch/pytorch#4029]
The pytorch version in my PC in 0.4, I compile source code to install it last time. But there is a server I have to use now, only has conda environment. And it's pytorch version is 0.2. So I meet some problems when I use softmax function. It said no dim keyword argument.

I want to do softmax on the third dimension of 3 dimension tensor (batch_size, m, n). 
That means
```
x = Variable(torch.rand(2, 3, 4))
F.softmax(x, dim=-1).
```
meet problems about dim argument.

How to solve this? or permute tensor to (n, batch_size, m), and do softmax(tensor)
```
x = x.permute(2, 0, 1)
F.softmax(x).permute(1, 2, 0)
```

**Comment by albanD** [comment]
Hi,

0.3 has been released now for conda and contains this feature.
The main website contains the information to install 0.3 with conda.
